saleor/graphql/utils.py

```diff
@@ -123,6 +123,31 @@ def filter_by_query_param(queryset, query, search_fields):
     return queryset
 
 
+def sort_queryset(queryset: QuerySet, sort_by: dict, sort_enum: SortInputObjectType):
```

Comment:
- return type: QuerySet
- sort_by is really a dict? you are accessing its `field` attr